cmd/pipeline/deploy.go

```diff
@@ -250,7 +250,8 @@ func (pc *Command) waitUntilRunning(ctx context.Context, name string, action *ty
 		defer wg.Done()
 		err := pc.streamPipelineLogs(waitCtx, name, action.Name)
 		if err != nil {
-			oktetoLog.Warning("there was an error streaming pipeline logs: %v", err)
+			oktetoLog.Warning("pipeline logs cannot be streamed due to connectivity issues")
+			oktetoLog.Infof("pipeline logs cannot be streamed due to connectivity issues: %v", err)
```

Comment:
This change and the one in `preview/deploy.go` are a couple of improvements in logs when there is some issue streaming logs. They were reverted with the resto of changes and I'm adding them back again